packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -315,6 +317,12 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     }
   }
 
+  a11yStatusTimeout: any
```

Comment:
```suggestion
  a11yStatusTimeout: number
```

We have conflicting definitions with `node.setTimeout`, https://github.com/Microsoft/TypeScript/issues/842
But we can easily solve it with usage `window.setTimeout`. Opened for better solutions

/cc @kuzhelov

Reply:
actually, would rather leave it as `any`, given that this is used as an opaque ref. Other options have more downsides than this, to my mind.

Speaking of `window`-based one, in particular - this would require us to make refactoring for the entire code base of the app. While it is a step to express the actual state of things we have at the moment, I would rather wait a bit with that, as in the upcoming future we might rework our codebase towards consuming `window` abstraction instead of direct DOM `window` access. Also, think that this is a pretty minor thing to worry about.

Please, folks, share your thoughts on that. @layershifter, @silviuavram